reflex/config.py

```diff
@@ -168,6 +168,9 @@ class Config(Base):
     # Additional frontend packages to install.
     frontend_packages: List[str] = []
 
+    # The bun path
+    bun_path: Optional[str] = constants.BUN_PATH
```

Comment:
```suggestion
    bun_path: str = constants.BUN_PATH
```

i think this is all that's need to fix the pre-commit errors